tests/unit/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Set Dummy Environments"""
```

Comment:
@nihit Running `pytest` without setting an environment errored out. I've added dummy API_KEY environments to handle it under `test/unit`. Not sure if this lines up with the way you guys want to export API_KEY for unit test cases.

If not let me know, I'll remove it.

Reply:
Thanks @Sardhendu, yep you'd wanna run `make test` which sets the environment variables for running the test. 

But this change works just fine too!

Reply:
Good callout though, we will add instructions to run the test in Contributing guidelines